MAUI/Shimmer/getting-started.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the getting started with Syncfusion .NET MAUI Shimmer (SfShimmer) control, its elements and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Getting Started of .NET MAUI Shimmer (SfShimmer)
+
+This section explains how to add the Shimmer control.
+
+## Creating an application using the .NET MAUI Shimmer
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfShimmer` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core), and then install it.
```

Comment:
2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfShimmer` to your project, open the NuGet package manager in Visual Studio, and search for the [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core), and then install it.

Reply:
Comments addressed